tests/android/app/build.gradle

```diff
@@ -51,6 +51,7 @@ android {
         targetSdkVersion flutter.targetSdkVersion
         versionCode flutterVersionCode.toInteger()
         versionName flutterVersionName
+        multiDexEnabled true
```

Comment:
I needed to use `multiDex` to fix the `flutter test` command. Is this a problem that I'm adding `multiDex`?

Output of the exception (without `multiDex`): https://github.com/nilsreichardt/flutterfire/runs/6877416338?check_suite_focus=true

Reply:
I think multidex is automatically added with minSdkVersion 21, do we want to keep the 19 version? The Android repartition is really low for this SDK 🤔

Reply:
+1 for bumping minSdkVersion to 21 on test apps, multidex has caused headaches for me before on other CI things (though this was a while back and on RNFirebase)

Reply:
Bumping the `minSdkVersion` to 21 in 622cd46f9894f22402ffc3e6c012a076b907e057 ✅